modules/_labs/select/react/lib/SelectOption.tsx

```diff
@@ -68,9 +77,11 @@ const Option = styled('li')<SelectOptionProps>(
       };
     } else {
       // ...else return hover and selected (via aria-selected) styles
+      const selectedBgColor = theme.canvas.palette.primary.lightest;
       const selectedStyles = {
         '&[aria-selected="true"]': {
-          backgroundColor: theme.canvas.palette.primary.lightest,
+          backgroundColor: selectedBgColor,
+          color: pickForegroundColor(selectedBgColor),
```

Comment:
@anicholls To Donovan's point in the slack conversation you shared, it does look a little odd for the text color in selected options (`blackPepper600`) to be slightly darker than the text color in non-selected options (`blackPepper300`). Subtle, though -- probably ok for now until we figure out the potential changes to the contrast logic.

Also, should I be using `pickForegroundColor` for the hover options? I'm guessing not since the hover bg color currently isn't themeable, but wanted to double-check.

Reply:
Yes I wouldn't worry about the `blackPepper300` vs. `blackPepper600` here. I'm having a larger conversation about how this works later this week.

Reply:
Sounds good.

Reply:
Actually, maybe for this case let's use `pickFoergroundColor(selectedBgColor, colors.blackPepper300)`.

I worry less about `blackPepper600` on a random theme color, but would prefer to use `blackPepper300` if it's dark on light with the canvas default theme.

Reply:
I've updated `pickForegroundColor` to use `colors.blackPepper300` for the selected option.